tests/unit/test_index.py

```diff
@@ -47,3 +55,9 @@ def test_render_index(index_html):
     expected_html, filenames = index_html
     html = Index(filenames).to_html()
     assert html == expected_html
+
+
+def test_filename_url():
```

Comment:
This test can be [parametrized](https://docs.pytest.org/en/6.2.x/parametrize.html#pytest-mark-parametrize).

Reply:
Done